src/components/InviteParticipants/InviteParticipants.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react'
+import InviteForm from './InviteForm'
+import Modal from '../Modal/Modal'
+import Button from '@material-ui/core/Button'
+import { selectToken } from '../../utils/selectors'
+import { useSelector } from 'react-redux'
+import { Api } from '../../services/api'
+
+const InviteParticipants = () => {
```

Comment:
in other files, I've seen exporting without default, maybe add a rules for one way or the others.

Reply:
took note in #303